src/frontend/pages/game.tsx

```diff
@@ -91,30 +101,51 @@ export default function Home(): ReactElement {
         <div className={styles.actionBar}>
           <div className={styles.actionBarLeft}>
             {/* TODO: both of these values will be given by an API/socket */}
-            <h4>You Are: </h4>
-            <h4>Current Coins: </h4>
+            {/* TODO: add some JS to change styling of the color based on what color it is  */}
+            <h4 className={styles.actionBarText}>
+              You Are: <span style={{ color: 'green' }}>{playerColor} </span>
+            </h4>
+            <h4 className={styles.actionBarText}>
+              Current Coins:{' '}
+              <span style={{ color: '#546ec9' }}>{playerCoins} </span>
+            </h4>
           </div>
           <div className={styles.actionBarMiddle}>
-            <button> Take</button>
-            {/* TODO: add socket send here */}
-          </div>
-          <div className={styles.actionBarRight}>
             <input
-              className={styles.inputBox}
               type="number"
               value={takeVal}
               onChange={(event) => {
                 inputOnChange(event.target.value);
               }}
               min="0"
```

Comment:
I would recommend we also use something like MIN_TAKE_VALUE for the minimum case